package-lock.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
There shouldn't be a package-lock.json, we're using yarn. Did you `npm install` something instead of `yarn add`ing it?